modules/form-field/css/package.json

```diff
@@ -17,7 +17,13 @@
   },
   "dependencies": {
     "@workday/canvas-kit-css-core": "^3.0.0-alpha.2",
-    "@workday/canvas-system-icons-web": "^1.0.1"
+    "@workday/canvas-kit-css-checkbox": "^3.0.0-alpha.0",
```

Comment:
These are only dev dependencies

Reply:
I'm not sure I understand, don't we need to reference them as dependencies to be able to use them?

Reply:
So right now we're re-exporting all of these in `modules/form-field/css/index.scss`. That's the old style though (where all form modules are available in this package). 

I'm about to push a commit that removes those re-exports. Since we don't use any of these for any of the exported code in `modules/form-field/css/lib`, they're not dependencies. However, since we are using them for demos in the storybook, we need to include them as devDependencies (also included in my commit).